Sources/StreamVideo/WebRTC/VideoCapturingUtils.swift

```diff
@@ -76,12 +78,19 @@ enum VideoCapturingUtils {
             if selectedFormat == nil {
                 selectedFormat = sortedFormats.first(where: { $0.dimensions.area >= preferredDimensions.area })
             }
+
+            // In case we cannot find a matching format based on the preferredDimensions
+            // we will choose the closest one.
+            if selectedFormat == nil {
+                selectedFormat = sortedFormats.min(by: { $0.diff < $1.diff })
```

Comment:
This is helping when the announced dimensions from the backend are bigger than the ones the device can offer